lib/commands/context.js

```diff
@@ -237,6 +248,155 @@ extensions.mobileGetContexts = async function mobileGetContexts () {
   }
 };
 
+commands.setWindow = async function setWindow (name, skipReadyCheck) {
+  try {
+    await this.setContext(name, _.noop, skipReadyCheck);
+  } catch (err) {
+    // translate the error in terms of windows
+    throw isErrorType(err, errors.NoSuchContextError)
+      ? new errors.NoSuchWindowError()
```

Comment:
should we keep the error message?

Reply:
No need. It is _always_ the default (see https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/context.js#L123) so it would be confusing to have.